src/parser/jobs/war/modules/Gauge.js

```diff
@@ -27,10 +27,16 @@ const RAGE_SPENDERS = {
 	[ACTIONS.DECIMATE.id]: 50,
 	[ACTIONS.UPHEAVAL.id]: 20,
 	[ACTIONS.ONSLAUGHT.id]: 20,
+	[ACTIONS.CHAOTIC_CYCLONE.id]:50,
+	[ACTIONS.INNER_CHAOS.id]:50,
 }
 
 // Actions that reduce Infuriate
 const INFURIATE_REDUCERS = [
+	ACTIONS.FELL_CLEAVE.id,
+	ACTIONS.INNER_BEAST.id,
+	ACTIONS.STEEL_CYCLONE.id,
```

Comment:
As mentioned before, irelevant to have Inner Beast and Steel Cyclone. We don't support logs that would have these in, since a trait upgrades them to Fell Cleave and Decimate, respectively. Otherwise, it looks good.